src/app/app.component.spec.ts

```diff
@@ -49,5 +56,11 @@ describe('AppComponent', () => {
         })
       })
     })
+
+    //outdated test placeholder til button updated
+    // describe('btnClick', () => {
+    //   it('Should ', async () => {
+    //   })
+    // })
```

Comment:
Will the button be updated?

Reply:
to be honest, i could probably remove the button, I will have a button with different functionality at some point. but currently in this feature the button will not have any functionality